components/automate-ui/src/app/page-components/feature-flags/feature-flags.component.scss

```diff
@@ -18,38 +18,38 @@
 
   .modal-content {
     margin: 15% auto;
-    background-color: $gray-pale;
-    padding: 20px;
-    border: 1px solid $gray7;
+    background-color: #FFF;
```

Comment:
This could actually use a wee bit of tidying up. `#FFF` and its equivalents  `$white` and `$chef-white` are all used throughout automate-ui. While we should standardize on just one of those three, that would touch a lot of files.
But standardizing on two is just a small change because `#FFF` is used in only a half dozen files. It is the least used of the three by far, so consider replacing those occurrences (and the one in L21 here).

Reply:
whoa, okay. im surprised we have a variable for white. but will take a look tomorrow.